rspace/src/test/scala/coop/rchain/rspace/nextgenrspace/history/HistoryGenerativeSpec.scala

```diff
@@ -32,30 +32,48 @@ class HistoryGenerativeSpec
   "history" should "accept new leafs (insert, update, delete)" in forAll(
     distinctListOf(arbitraryInsertAction)
   ) { actions: List[Data] =>
-    val emptyHistory = HistoryInstances.merging[Task](emptyRootHash, inMemHistoryStore)
+    val emptyMergingHistory = HistoryInstances.merging[Task](emptyRootHash, inMemHistoryStore)
```

Comment:
This test would be more readable if it tested one of the implementations at a time. Each of the implementations could be passed in using `TableDrivenPropertyChecks`

Reply:
I don't want to test both. I want to test the merging one against the one that does not merge.